ui/controls/chart.reel/chart.js

```diff
@@ -199,27 +199,29 @@ exports.Chart = Component.specialize(/** @lends Chart# */ {
 
     _addDisplayedSeries: {
         value: function() {
-            var legendsContainer = this._legend.content()[0][0],
-                legendElements = legendsContainer.getElementsByClassName('legend-entry'),
-                disabledSeries = this._seriesList.filter(function(x) { return x.disabled; }).map(function(x) { return x.key; }),
-                legendElement, series,
-                datasets = [],
-                i, length;
-            for (i = 0, length = legendElements.length; i < length; i++) {
-                legendElement = legendElements.item(i);
-                if (disabledSeries.length > 0 && disabledSeries.indexOf(d3.select(legendElement).datum()) != -1) {
-                    legendElement.classList.add('is-disabled');
-                } else {
-                    legendElement.classList.remove('is-disabled');
+            var legendsContainer = this._legend.content()[0][0];
+            if (legendsContainer && legendsContainer.childNodes && legendsContainer.childNodes[0]) {
+                var legendElements = legendsContainer.childNodes[0].childNodes,
```

Comment:
If there is a legend, yes it always has the same structure (as long as plottable.js doesn't change it...).
If you have a better way to avoid using `getElementsByClassName`, I'd be more than happy to avoid relying on the DOM structure to find the children.